### PR TITLE
Pre-release merge for docs 0.6.0

### PR DESCRIPTION
**Pull request by bmaltzan**

### CODE REVIEW

File: help/authority.md [hunk 1]
```diff
@@ -28,30 +28,22 @@ recommend that you share the paper password with all of the authors of
...
+If a paper is on your list of papers you own, but you are not registered as an author,
+you can use the [Change Author Status form](/auth/change-author-status).
+
+## For other authors
+
+Since most papers have multiple authors, it is quite likely that you are the author of a paper that you did not submit personally. 
+
+### To claim a paper
+
+If the submitter has provided you with the paper password you can use the [the Claim Ownership with a password form](/auth/need-paper-password).
```
Comment (SBBCornell) [inline help/authority.md:44]:
```suggestion
If the submitter has provided you with the paper password you can use the [Claim Ownership with a password form](/auth/need-paper-password).
```

Comment (SBBCornell) [reply help/authority.md:44]: removed extra 'the'

File: help/authority.md [hunk 2]
```diff
@@ -28,30 +28,22 @@ recommend that you share the paper password with all of the authors of
...
+
+If you do not have the paper password you can use the [the Claim Authorship form](/auth/request-ownership). Note that this process involves verification by our staff and may take a couple of days.
```
Comment (SBBCornell) [inline help/authority.md:46]:
```suggestion
If you do not have the paper password you can use the [Claim Authorship form](/auth/request-ownership). Note that this process involves verification by our staff and may take a couple of days.
```

Comment (SBBCornell) [reply help/authority.md:46]: removed extra 'the'

File: help/authority.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ support third-party submission by approved conference organizers and
 journals. Users may log-in to see [a list](/user/) of
```
Comment (SBBCornell) [inline help/authority.md:14]: this link could be more a11y friendly, like 'a list of papers'

File: help/authority.md [hunk 4]
```diff
@@ -28,30 +28,22 @@ recommend that you share the paper password with all of the authors of
 each paper you submit and encourage them to register themselves as
 owners.
 
-If you have submitted a paper in the past, it is quite likely that you
-are already registered as an author. However, because this is an
-imprecise process of matching e-mail addresses and names, we have
-probably made mistakes in some cases. If papers you have submitted are
+If papers you have submitted are
 missing entirely from your [list](/user/) of papers, you
```
Comment (SBBCornell) [inline help/authority.md:32]:
```suggestion
missing entirely from your [list of papers](/user/), you
```

**Comment by jimentwood** [comment]
Thanks @SBBCornell - changes applied

**Pull request merged**